openmmtools/multistate/multistateanalyzer.py

```diff
@@ -1915,11 +1920,13 @@ def _compute_free_energy(self):
         logger.debug("Computing covariance matrix...")
 
         try:
-            # pymbar 2
-            (Deltaf_ij, dDeltaf_ij) = self.mbar.getFreeEnergyDifferences()
-        except ValueError:
             # pymbar 3
             (Deltaf_ij, dDeltaf_ij, _) = self.mbar.getFreeEnergyDifferences()
```

Comment:
This should fix the failing tests.
```suggestion
            Deltaf_ij, dDeltaf_ij = self.mbar.getFreeEnergyDifferences()
```
pymbar 3 actually returns two things by default, unless we specify `return_theta=True`.